content/md/en/docs/build/randomness.md

```diff
@@ -15,10 +15,10 @@ In traditional computer applications, what people call random numbers are actual
 Pseudo-randomness depends on a sufficiently random seed—provided by the user or by an external source—and an algorithm that manipulates the seed to generate a series of seemingly-random numbers. 
 The result is a pseudo-random number that is difficult to predict because of the algorithm used to generate it, but deterministic because the same seed input always produces the same series of output.
 
-However, applications that run on the blockchain are more tightly constrained because all authorities in the network must agree on any on-chain value, including any randomness data that is injected.
-Because of this constraint, you can't use real randomness directly in blockchain applications.
+However, applications that run on the blockchain are more tightly constrained. 
+Pseudo-randomness introduces a security vulnerability for situations where being able to predict the outcome of a random function can lead to gaming the system.
```

Comment:
```suggestion
However, applications that run on the blockchain are more tightly constrained because all authorities in the network must agree on any on-chain value, including any randomness data that is injected.
Because of this constraint, you can't use real randomness directly in blockchain applications.
For this reason, using randomness on-chain securely must utilize techniques such as VRFs otherwise it becomes possible to predict the outcome of a random function which could lead to gaming the system.
```